.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = false
+
+[*.go]
+indent_style = tab
+tab_width = 4
```

Comment:
**Question:** this comes on top/separately of gofmt, and I believe it requires installing an extension in VSCode. Do you think this will result in the opposite of what it's supposed to do: different formatting?